tests/tests_api/test_teacher_under_18.py

```diff
@@ -11,9 +11,11 @@ def test_teacher_under_18_create(api_client, faker):
 
     data = {
         "personal_info": personal_info.id,
-        "additional_skills_comment": faker.pystr(max_chars=DEFAULT_CHOICE_CHAR_FIELD_MAX_LENGTH),
-        "can_help_with_speaking_club": faker.pybool(),
+        "can_host_speaking_club": faker.pybool(),
         "comment": faker.text(),
+        "status_since": faker.date_time(tzinfo=tz.gettz("UTC")),
```

Comment:
I think more standard approach is to use pytz lib, because it gives us some constants, which is better than strings

Reply:
Done